app_llm.py

```diff
@@ -5,12 +5,12 @@
 
 import requests
 import streamlit as st
+from simpleeval import simple_eval
 
 import msfocr.data.dhis2
-import msfocr.doctr.ocr_functions
+import msfocr.docTR.ocr_functions
```

Comment:
I changed the package names to match [pep8](https://pep8.org/#package-and-module-names) when I put together the code for the demo last week. You might need to to clear the cache files (remove the `__pycache__` folders) and reinstall if your editor is still suggesting the old package names.  
```suggestion
import msfocr.doctr.ocr_functions
```